public/editor/scripts/editor/js/bramble-editor.js

```diff
@@ -14,7 +14,13 @@ define(function(require) {
       // Start loading the Bramble editor resources
       Bramble.load("#webmaker-bramble",{
         url: options.editorUrl,
-        hideUntilReady: true
+        hideUntilReady: true,
+        extensions: {
+          enable: [
+            // Watch and warn users on rename or delete of index.html
+            'bramble-watch-index.html'
```

Comment:
The error comes from Brackets!

I think the main problem is that we allow you to enable extensions and determine locale using the Bramble API (kinda like what you've done above) which means we can do query param parsing on the client and pass those to Brackets via the Bramble API. However, in Brackets, in a bunch of places, we read directly from the query string that has been passed in through the bramble url and we take advantage of this [here](https://github.com/mozilla/thimble.mozilla.org/blob/master/server/routes/main/editor.js#L82) to inject those query params that were passed into the thimble url directly into the bramble url, and our client API can't handle both (injecting options via the query params AND passing options via the API)